src/commands/dev.ts

```diff
@@ -260,13 +260,13 @@ export async function command(commandOptions: CommandOptions) {
         id: fileLoc,
         data: missingWebModule,
       });
-      const isHmrEnabled = builtFileResult.result.includes('import.meta.hot');
```

Comment:
1 thing missing here: `accept()` is not the only way to be HMR enabled. You can have a `decline()` call or a `dispose` call that should all be handled even if no accept call exists.

I'm not saying that we need to implement all of that now, but I do think we should track this as two separate pieces of state now: 
- `isHmrEnabled`: only send update events for these files, to be handled by the client
- `isAccept`: don't bubble events past these nodes in the tree.

Reply:
Hmm should we convert that into an issue?

Reply:
This isn't an issue so much as the spec as defined in https://github.com/pikapkg/esm-hmr (although it could probably be called out more explicitly).